docs/en/02_Developer_Guides/01_Templates/05_Template_Inheritance.md

```diff
@@ -43,20 +43,59 @@ The manifest is created whenever you flush your SilverStripe cache by appending
 example by visiting `http://yoursite.com/?flush=1`. When your include the `flush=1` flag, the manifest class will search 
 your entire project for the appropriate `.ss` files located in `template` directory and save that information for later.
 
-It will each and prioritize templates in the following priority:
-
-1. mysite (or other name given to site folder)
-2. module-specific themes (e.g. themes/simple_blog)
-3. themes (e.g. themes/simple)
-4. modules  (e.g. blog)
-5. framework
-
 <div class="warning">
 Whenever you add or remove template files, rebuild the manifest by visiting `http://yoursite.com/?flush=1`. You can 
 flush the cache from any page, (.com/home?flush=1, .com/admin?flush=1, etc.). Flushing the cache can be slow, so you 
 only need to do it when you're developing new templates.
 </div>
 
+## Template Priority
```

Comment:
@unclecheese I think you need to fix up the docs in a similar way in `i18n.md`. Can you please also check `Directory_Structure.md` and ensure those instructions still make sense?

Is there no need for mentioning this in the upgrade guide? Will template and i18n inheritance work like in 3.x by default?